library/src/main/java/org/codice/security/saml/EntityInformation.java

```diff
@@ -94,45 +90,38 @@ Binding getBinding(AuthnRequest request, Binding preferred) {
     return preferred != null ? preferred : PREFERRED_BINDING;
   }
 
-  public Map<Binding, String> getAssertionConsumerServicesURLs() {
-    return assertionConsumerServices
-        .entrySet()
-        .stream()
-        .map(entry -> new AbstractMap.SimpleEntry<>(entry.getKey(), entry.getKey().getUri()))
-        .collect(
-            Collectors.toMap(AbstractMap.SimpleEntry::getKey, AbstractMap.SimpleEntry::getValue));
-  }
-
   public ServiceInfo getAssertionConsumerService(Binding binding) {
     return assertionConsumerServices.get(binding);
   }
 
   public ServiceInfo getAssertionConsumerService(
       AuthnRequest request, Binding preferred, Integer index) {
     ServiceInfo si = null;
```

Comment:
✏️ Super nitpick: we don't use acronym variable names a lot in the codebase. Probably more consistent to change `si` to `serviceInfo` or something of that manner instead.

Reply:
I think I'd rather leave this as it is in DDF.